src/imitation/scripts/parallel.py

```diff
@@ -0,0 +1,66 @@
+from typing import Callable
+
+import ray
+import ray.tune
+
+from imitation.scripts.config.parallel import parallel_ex
+import imitation.util as util
+
+
+@parallel_ex.main
+def parallel(inner_experiment_name: str, search_space: dict) -> None:
```

Comment:
How do I set fixed parameter values (that differ from the defaults for the experiment)? Do I have to make it part of `search_space` (even though I'm not searching over it)?

Reply:
Yes, I think that's right

Reply:
This is a little unfortunate as it's often useful to distinction between fixed and variable parameters, as you might e.g. want to report the variable parameters as the TB run name but not the fixed ones. I'd suggest considering restructuring this to distinguish them, but if it's not causing you problems feel free to ignore.

Reply:
Sounds good, I'll do that if it makes TB too hard to read later.

Reply:
Implemented in #101